libraries/botbuilder-ai/src/luisRecognizer.ts

```diff
@@ -197,10 +197,8 @@ export class LuisRecognizer {
             return this.luisClient.prediction.resolve(
                 this.application.applicationId, utterance,
                 {
-                    timezoneOffset: this.options.timezoneOffset,
-                    verbose: this.options.includeAllIntents,
```

Comment:
In the old version they pass `this.options.includeAllIntents` to `verbose`. In the object spread version this will be mapped to `includeAllIntents`. 

Is this still correct?

Reply:
Good catch!

Reply:
I'll go back in and make sure everything is mapped correctly. Thanks for catching this!